src/cloud_provider/aws/application.rs

```diff
@@ -20,7 +20,7 @@ use crate::logger::Logger;
 use crate::models::{Context, Listen, Listener, Listeners, ListenersHelper, Port};
 use ::function_name::named;
 
-pub struct Application {
+pub struct ApplicationAws {
```

Comment:
Out of curiosity, what's the reason adding provider prefix?

Reply:
To avoid confusion, having 4 times the same struct (1 for each provider + 1 for the trait) make it really hard to know where you are when just looking at the code.
It avoids also the silly conflicting imports